2014.06.18 16:30:26 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性
2014.06.18 16:30:26 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:30:26 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-getMemberId-service-unavailable","message":"调用后端服务member.getMemberId抛异常:org.hibernate.exception.GenericJDBCException，服务不可用。\n异常信息:\norg.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.ClientAppkeyResolver$$EnhancerByCGLIB$$517fd9f7.findAllClientAppkey(<generated>)\r\n\tat com.xtrainning.hop.HopAppSecretManager.loadAppKey(HopAppSecretManager.java:46)\r\n\tat com.xtrainning.hop.HopAppSecretManager.getSecret(HopAppSecretManager.java:40)\r\n\tat com.xtrainning.hop.HopAppSecretManager.isValidAppKey(HopAppSecretManager.java:34)\r\n\tat com.rop.security.DefaultSecurityManager.validateSystemParameters(DefaultSecurityManager.java:67)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:329)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 19 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot load JDBC driver class 'com.mysql.jdbc.Driver'\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1136)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:880)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 24 more\r\nCaused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver\r\n\tat java.net.URLClassLoader$1.run(URLClassLoader.java:202)\r\n\tat java.security.AccessController.doPrivileged(Native Method)\r\n\tat java.net.URLClassLoader.findClass(URLClassLoader.java:190)\r\n\tat java.lang.ClassLoader.loadClass(ClassLoader.java:306)\r\n\tat java.lang.ClassLoader.loadClass(ClassLoader.java:247)\r\n\tat java.lang.Class.forName0(Native Method)\r\n\tat java.lang.Class.forName(Class.java:169)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1130)\r\n\t... 27 more\r\n"}]}
2014.06.18 16:30:26 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性
2014.06.18 16:30:26 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:30:26 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-login-service-unavailable","message":"调用后端服务member.login抛异常:org.hibernate.exception.GenericJDBCException，服务不可用。\n异常信息:\norg.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.ClientAppkeyResolver$$EnhancerByCGLIB$$517fd9f7.findAllClientAppkey(<generated>)\r\n\tat com.xtrainning.hop.HopAppSecretManager.loadAppKey(HopAppSecretManager.java:46)\r\n\tat com.xtrainning.hop.HopAppSecretManager.getSecret(HopAppSecretManager.java:40)\r\n\tat com.xtrainning.hop.HopAppSecretManager.isValidAppKey(HopAppSecretManager.java:34)\r\n\tat com.rop.security.DefaultSecurityManager.validateSystemParameters(DefaultSecurityManager.java:67)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:329)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 19 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot load JDBC driver class 'com.mysql.jdbc.Driver'\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1136)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:880)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 24 more\r\nCaused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver\r\n\tat java.net.URLClassLoader$1.run(URLClassLoader.java:202)\r\n\tat java.security.AccessController.doPrivileged(Native Method)\r\n\tat java.net.URLClassLoader.findClass(URLClassLoader.java:190)\r\n\tat java.lang.ClassLoader.loadClass(ClassLoader.java:306)\r\n\tat java.lang.ClassLoader.loadClass(ClassLoader.java:247)\r\n\tat java.lang.Class.forName0(Native Method)\r\n\tat java.lang.Class.forName(Class.java:169)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1130)\r\n\t... 27 more\r\n"}]}
2014.06.18 16:30:26 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性
2014.06.18 16:30:26 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:30:26 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-getProfile-service-unavailable","message":"调用后端服务member.getProfile抛异常:org.hibernate.exception.GenericJDBCException，服务不可用。\n异常信息:\norg.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.ClientAppkeyResolver$$EnhancerByCGLIB$$517fd9f7.findAllClientAppkey(<generated>)\r\n\tat com.xtrainning.hop.HopAppSecretManager.loadAppKey(HopAppSecretManager.java:46)\r\n\tat com.xtrainning.hop.HopAppSecretManager.getSecret(HopAppSecretManager.java:40)\r\n\tat com.xtrainning.hop.HopAppSecretManager.isValidAppKey(HopAppSecretManager.java:34)\r\n\tat com.rop.security.DefaultSecurityManager.validateSystemParameters(DefaultSecurityManager.java:67)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:329)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 19 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot load JDBC driver class 'com.mysql.jdbc.Driver'\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1136)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:880)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 24 more\r\nCaused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver\r\n\tat java.net.URLClassLoader$1.run(URLClassLoader.java:202)\r\n\tat java.security.AccessController.doPrivileged(Native Method)\r\n\tat java.net.URLClassLoader.findClass(URLClassLoader.java:190)\r\n\tat java.lang.ClassLoader.loadClass(ClassLoader.java:306)\r\n\tat java.lang.ClassLoader.loadClass(ClassLoader.java:247)\r\n\tat java.lang.Class.forName0(Native Method)\r\n\tat java.lang.Class.forName(Class.java:169)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1130)\r\n\t... 27 more\r\n"}]}{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-getProfile-service-unavailable","message":"调用后端服务member.getProfile抛异常:org.springframework.transaction.CannotCreateTransactionException，服务不可用。\n异常信息:\njava.util.concurrent.ExecutionException: org.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat java.util.concurrent.FutureTask$Sync.innerGet(FutureTask.java:232)\r\n\tat java.util.concurrent.FutureTask.get(FutureTask.java:91)\r\n\tat com.rop.impl.AnnotationServletServiceRouter.service(AnnotationServletServiceRouter.java:112)\r\n\tat com.rop.RopServlet.service(RopServlet.java:54)\r\n\tat javax.servlet.http.HttpServlet.service(HttpServlet.java:803)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)\r\n\tat org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:88)\r\n\tat org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)\r\n\tat org.springframework.orm.hibernate3.support.OpenSessionInViewFilter.doFilterInternal(OpenSessionInViewFilter.java:198)\r\n\tat org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)\r\n\tat org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:230)\r\n\tat org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)\r\n\tat org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)\r\n\tat org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:104)\r\n\tat org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)\r\n\tat org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:261)\r\n\tat org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)\r\n\tat org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:581)\r\n\tat org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.SysSessionResolver$$EnhancerByCGLIB$$8c9ae8b7.getValidSessionBySessionId(<generated>)\r\n\tat com.xtrainning.hop.HopSessionManager.getSession(HopSessionManager.java:44)\r\n\tat com.rop.impl.SimpleRopRequestContext.getSession(SimpleRopRequestContext.java:142)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:373)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\t... 1 more\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 17 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot load JDBC driver class 'com.mysql.jdbc.Driver'\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1136)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:880)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 22 more\r\nCaused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver\r\n\tat java.net.URLClassLoader$1.run(URLClassLoader.java:202)\r\n\tat java.security.AccessController.doPrivileged(Native Method)\r\n\tat java.net.URLClassLoader.findClass(URLClassLoader.java:190)\r\n\tat java.lang.ClassLoader.loadClass(ClassLoader.java:306)\r\n\tat java.lang.ClassLoader.loadClass(ClassLoader.java:247)\r\n\tat java.lang.Class.forName0(Native Method)\r\n\tat java.lang.Class.forName(Class.java:169)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1130)\r\n\t... 25 more\r\n"}]}
2014.06.18 16:31:55 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性
2014.06.18 16:31:55 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:31:55 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-getMemberId-service-unavailable","message":"调用后端服务member.getMemberId抛异常:org.hibernate.exception.GenericJDBCException，服务不可用。\n异常信息:\norg.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.ClientAppkeyResolver$$EnhancerByCGLIB$$a11885a6.findAllClientAppkey(<generated>)\r\n\tat com.xtrainning.hop.HopAppSecretManager.loadAppKey(HopAppSecretManager.java:46)\r\n\tat com.xtrainning.hop.HopAppSecretManager.getSecret(HopAppSecretManager.java:40)\r\n\tat com.xtrainning.hop.HopAppSecretManager.isValidAppKey(HopAppSecretManager.java:34)\r\n\tat com.rop.security.DefaultSecurityManager.validateSystemParameters(DefaultSecurityManager.java:67)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:329)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 19 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 24 more\r\nCaused by: java.sql.SQLException: The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1074)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:988)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:974)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:919)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.validateStringValues(ConnectionPropertiesImpl.java:311)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$BooleanConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:99)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:210)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl.initializeProperties(ConnectionPropertiesImpl.java:2792)\r\n\tat com.mysql.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:3396)\r\n\tat com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:807)\r\n\tat com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)\r\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)\r\n\tat java.lang.reflect.Constructor.newInstance(Constructor.java:513)\r\n\tat com.mysql.jdbc.Util.handleNewInstance(Util.java:411)\r\n\tat com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)\r\n\tat com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:334)\r\n\tat org.apache.tomcat.dbcp.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)\r\n\tat org.apache.tomcat.dbcp.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)\r\n\t... 28 more\r\n"}]}
2014.06.18 16:31:55 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性
2014.06.18 16:31:55 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:31:55 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-login-service-unavailable","message":"调用后端服务member.login抛异常:org.hibernate.exception.GenericJDBCException，服务不可用。\n异常信息:\norg.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.ClientAppkeyResolver$$EnhancerByCGLIB$$a11885a6.findAllClientAppkey(<generated>)\r\n\tat com.xtrainning.hop.HopAppSecretManager.loadAppKey(HopAppSecretManager.java:46)\r\n\tat com.xtrainning.hop.HopAppSecretManager.getSecret(HopAppSecretManager.java:40)\r\n\tat com.xtrainning.hop.HopAppSecretManager.isValidAppKey(HopAppSecretManager.java:34)\r\n\tat com.rop.security.DefaultSecurityManager.validateSystemParameters(DefaultSecurityManager.java:67)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:329)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 19 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 24 more\r\nCaused by: java.sql.SQLException: The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1074)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:988)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:974)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:919)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.validateStringValues(ConnectionPropertiesImpl.java:311)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$BooleanConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:99)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:210)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl.initializeProperties(ConnectionPropertiesImpl.java:2792)\r\n\tat com.mysql.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:3396)\r\n\tat com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:807)\r\n\tat com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)\r\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)\r\n\tat java.lang.reflect.Constructor.newInstance(Constructor.java:513)\r\n\tat com.mysql.jdbc.Util.handleNewInstance(Util.java:411)\r\n\tat com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)\r\n\tat com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:334)\r\n\tat org.apache.tomcat.dbcp.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)\r\n\tat org.apache.tomcat.dbcp.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)\r\n\t... 28 more\r\n"}]}
2014.06.18 16:31:55 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性
2014.06.18 16:31:55 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:31:55 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-getProfile-service-unavailable","message":"调用后端服务member.getProfile抛异常:org.hibernate.exception.GenericJDBCException，服务不可用。\n异常信息:\norg.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.ClientAppkeyResolver$$EnhancerByCGLIB$$a11885a6.findAllClientAppkey(<generated>)\r\n\tat com.xtrainning.hop.HopAppSecretManager.loadAppKey(HopAppSecretManager.java:46)\r\n\tat com.xtrainning.hop.HopAppSecretManager.getSecret(HopAppSecretManager.java:40)\r\n\tat com.xtrainning.hop.HopAppSecretManager.isValidAppKey(HopAppSecretManager.java:34)\r\n\tat com.rop.security.DefaultSecurityManager.validateSystemParameters(DefaultSecurityManager.java:67)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:329)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 19 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 24 more\r\nCaused by: java.sql.SQLException: The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1074)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:988)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:974)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:919)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.validateStringValues(ConnectionPropertiesImpl.java:311)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$BooleanConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:99)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:210)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl.initializeProperties(ConnectionPropertiesImpl.java:2792)\r\n\tat com.mysql.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:3396)\r\n\tat com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:807)\r\n\tat com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)\r\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)\r\n\tat java.lang.reflect.Constructor.newInstance(Constructor.java:513)\r\n\tat com.mysql.jdbc.Util.handleNewInstance(Util.java:411)\r\n\tat com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)\r\n\tat com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:334)\r\n\tat org.apache.tomcat.dbcp.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)\r\n\tat org.apache.tomcat.dbcp.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)\r\n\t... 28 more\r\n"}]}{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-getProfile-service-unavailable","message":"调用后端服务member.getProfile抛异常:org.springframework.transaction.CannotCreateTransactionException，服务不可用。\n异常信息:\njava.util.concurrent.ExecutionException: org.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat java.util.concurrent.FutureTask$Sync.innerGet(FutureTask.java:232)\r\n\tat java.util.concurrent.FutureTask.get(FutureTask.java:91)\r\n\tat com.rop.impl.AnnotationServletServiceRouter.service(AnnotationServletServiceRouter.java:112)\r\n\tat com.rop.RopServlet.service(RopServlet.java:54)\r\n\tat javax.servlet.http.HttpServlet.service(HttpServlet.java:723)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)\r\n\tat org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:88)\r\n\tat org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)\r\n\tat org.springframework.orm.hibernate3.support.OpenSessionInViewFilter.doFilterInternal(OpenSessionInViewFilter.java:198)\r\n\tat org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)\r\n\tat org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)\r\n\tat org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)\r\n\tat org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)\r\n\tat org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)\r\n\tat org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)\r\n\tat org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:293)\r\n\tat org.apache.coyote.http11.Http11AprProcessor.process(Http11AprProcessor.java:879)\r\n\tat org.apache.coyote.http11.Http11AprProtocol$Http11ConnectionHandler.process(Http11AprProtocol.java:617)\r\n\tat org.apache.tomcat.util.net.AprEndpoint$Worker.run(AprEndpoint.java:1760)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.SysSessionResolver$$EnhancerByCGLIB$$dc339466.getValidSessionBySessionId(<generated>)\r\n\tat com.xtrainning.hop.HopSessionManager.getSession(HopSessionManager.java:44)\r\n\tat com.rop.impl.SimpleRopRequestContext.getSession(SimpleRopRequestContext.java:142)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:373)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\t... 1 more\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 17 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 22 more\r\nCaused by: java.sql.SQLException: The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1074)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:988)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:974)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:919)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.validateStringValues(ConnectionPropertiesImpl.java:311)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$BooleanConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:99)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:210)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl.initializeProperties(ConnectionPropertiesImpl.java:2792)\r\n\tat com.mysql.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:3396)\r\n\tat com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:807)\r\n\tat com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)\r\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)\r\n\tat java.lang.reflect.Constructor.newInstance(Constructor.java:513)\r\n\tat com.mysql.jdbc.Util.handleNewInstance(Util.java:411)\r\n\tat com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)\r\n\tat com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:334)\r\n\tat org.apache.tomcat.dbcp.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)\r\n\tat org.apache.tomcat.dbcp.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)\r\n\t... 26 more\r\n"}]}
2014.06.18 16:33:37 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性
2014.06.18 16:33:37 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:33:37 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-getMemberId-service-unavailable","message":"调用后端服务member.getMemberId抛异常:org.hibernate.exception.GenericJDBCException，服务不可用。\n异常信息:\norg.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.ClientAppkeyResolver$$EnhancerByCGLIB$$a02d8dce.findAllClientAppkey(<generated>)\r\n\tat com.xtrainning.hop.HopAppSecretManager.loadAppKey(HopAppSecretManager.java:46)\r\n\tat com.xtrainning.hop.HopAppSecretManager.getSecret(HopAppSecretManager.java:40)\r\n\tat com.xtrainning.hop.HopAppSecretManager.isValidAppKey(HopAppSecretManager.java:34)\r\n\tat com.rop.security.DefaultSecurityManager.validateSystemParameters(DefaultSecurityManager.java:67)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:329)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 19 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 24 more\r\nCaused by: java.sql.SQLException: The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1074)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:988)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:974)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:919)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.validateStringValues(ConnectionPropertiesImpl.java:311)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$BooleanConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:99)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:210)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl.initializeProperties(ConnectionPropertiesImpl.java:2792)\r\n\tat com.mysql.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:3396)\r\n\tat com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:807)\r\n\tat com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)\r\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)\r\n\tat java.lang.reflect.Constructor.newInstance(Constructor.java:513)\r\n\tat com.mysql.jdbc.Util.handleNewInstance(Util.java:411)\r\n\tat com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)\r\n\tat com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:334)\r\n\tat org.apache.tomcat.dbcp.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)\r\n\tat org.apache.tomcat.dbcp.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)\r\n\t... 28 more\r\n"}]}
2014.06.18 16:33:38 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性
2014.06.18 16:33:38 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:33:38 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-login-service-unavailable","message":"调用后端服务member.login抛异常:org.hibernate.exception.GenericJDBCException，服务不可用。\n异常信息:\norg.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.ClientAppkeyResolver$$EnhancerByCGLIB$$a02d8dce.findAllClientAppkey(<generated>)\r\n\tat com.xtrainning.hop.HopAppSecretManager.loadAppKey(HopAppSecretManager.java:46)\r\n\tat com.xtrainning.hop.HopAppSecretManager.getSecret(HopAppSecretManager.java:40)\r\n\tat com.xtrainning.hop.HopAppSecretManager.isValidAppKey(HopAppSecretManager.java:34)\r\n\tat com.rop.security.DefaultSecurityManager.validateSystemParameters(DefaultSecurityManager.java:67)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:329)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 19 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 24 more\r\nCaused by: java.sql.SQLException: The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1074)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:988)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:974)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:919)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.validateStringValues(ConnectionPropertiesImpl.java:311)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$BooleanConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:99)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:210)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl.initializeProperties(ConnectionPropertiesImpl.java:2792)\r\n\tat com.mysql.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:3396)\r\n\tat com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:807)\r\n\tat com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)\r\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)\r\n\tat java.lang.reflect.Constructor.newInstance(Constructor.java:513)\r\n\tat com.mysql.jdbc.Util.handleNewInstance(Util.java:411)\r\n\tat com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)\r\n\tat com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:334)\r\n\tat org.apache.tomcat.dbcp.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)\r\n\tat org.apache.tomcat.dbcp.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)\r\n\t... 28 more\r\n"}]}
2014.06.18 16:33:38 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性
2014.06.18 16:33:38 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:33:38 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-getProfile-service-unavailable","message":"调用后端服务member.getProfile抛异常:org.hibernate.exception.GenericJDBCException，服务不可用。\n异常信息:\norg.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.ClientAppkeyResolver$$EnhancerByCGLIB$$a02d8dce.findAllClientAppkey(<generated>)\r\n\tat com.xtrainning.hop.HopAppSecretManager.loadAppKey(HopAppSecretManager.java:46)\r\n\tat com.xtrainning.hop.HopAppSecretManager.getSecret(HopAppSecretManager.java:40)\r\n\tat com.xtrainning.hop.HopAppSecretManager.isValidAppKey(HopAppSecretManager.java:34)\r\n\tat com.rop.security.DefaultSecurityManager.validateSystemParameters(DefaultSecurityManager.java:67)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:329)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 19 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 24 more\r\nCaused by: java.sql.SQLException: The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1074)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:988)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:974)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:919)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.validateStringValues(ConnectionPropertiesImpl.java:311)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$BooleanConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:99)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:210)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl.initializeProperties(ConnectionPropertiesImpl.java:2792)\r\n\tat com.mysql.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:3396)\r\n\tat com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:807)\r\n\tat com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)\r\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)\r\n\tat java.lang.reflect.Constructor.newInstance(Constructor.java:513)\r\n\tat com.mysql.jdbc.Util.handleNewInstance(Util.java:411)\r\n\tat com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)\r\n\tat com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:334)\r\n\tat org.apache.tomcat.dbcp.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)\r\n\tat org.apache.tomcat.dbcp.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)\r\n\t... 28 more\r\n"}]}{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-getProfile-service-unavailable","message":"调用后端服务member.getProfile抛异常:org.springframework.transaction.CannotCreateTransactionException，服务不可用。\n异常信息:\njava.util.concurrent.ExecutionException: org.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat java.util.concurrent.FutureTask$Sync.innerGet(FutureTask.java:232)\r\n\tat java.util.concurrent.FutureTask.get(FutureTask.java:91)\r\n\tat com.rop.impl.AnnotationServletServiceRouter.service(AnnotationServletServiceRouter.java:112)\r\n\tat com.rop.RopServlet.service(RopServlet.java:54)\r\n\tat javax.servlet.http.HttpServlet.service(HttpServlet.java:723)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)\r\n\tat org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:88)\r\n\tat org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)\r\n\tat org.springframework.orm.hibernate3.support.OpenSessionInViewFilter.doFilterInternal(OpenSessionInViewFilter.java:198)\r\n\tat org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)\r\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)\r\n\tat org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)\r\n\tat org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)\r\n\tat org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)\r\n\tat org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)\r\n\tat org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)\r\n\tat org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:293)\r\n\tat org.apache.coyote.http11.Http11AprProcessor.process(Http11AprProcessor.java:879)\r\n\tat org.apache.coyote.http11.Http11AprProtocol$Http11ConnectionHandler.process(Http11AprProtocol.java:617)\r\n\tat org.apache.tomcat.util.net.AprEndpoint$Worker.run(AprEndpoint.java:1760)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.springframework.transaction.CannotCreateTransactionException: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:596)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:371)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:335)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:105)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.SysSessionResolver$$EnhancerByCGLIB$$db489c8e.getValidSessionBySessionId(<generated>)\r\n\tat com.xtrainning.hop.HopSessionManager.getSession(HopSessionManager.java:44)\r\n\tat com.rop.impl.SimpleRopRequestContext.getSession(SimpleRopRequestContext.java:142)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:373)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\t... 1 more\r\nCaused by: org.hibernate.exception.GenericJDBCException: Cannot open connection\r\n\tat org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:126)\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:114)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)\r\n\tat org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)\r\n\tat org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:142)\r\n\tat org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:85)\r\n\tat org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1353)\r\n\tat org.springframework.orm.hibernate3.HibernateTransactionManager.doBegin(HibernateTransactionManager.java:555)\r\n\t... 17 more\r\nCaused by: org.apache.tomcat.dbcp.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)\r\n\tat org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)\r\n\tat org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)\r\n\t... 22 more\r\nCaused by: java.sql.SQLException: The connection property 'useUnicode' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'true;characterEncoding=utf-8;autoReconnect=true' is not in this set.\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1074)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:988)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:974)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:919)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.validateStringValues(ConnectionPropertiesImpl.java:311)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$BooleanConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:99)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl$ConnectionProperty.initializeFrom(ConnectionPropertiesImpl.java:210)\r\n\tat com.mysql.jdbc.ConnectionPropertiesImpl.initializeProperties(ConnectionPropertiesImpl.java:2792)\r\n\tat com.mysql.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:3396)\r\n\tat com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:807)\r\n\tat com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)\r\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)\r\n\tat java.lang.reflect.Constructor.newInstance(Constructor.java:513)\r\n\tat com.mysql.jdbc.Util.handleNewInstance(Util.java:411)\r\n\tat com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)\r\n\tat com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:334)\r\n\tat org.apache.tomcat.dbcp.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)\r\n\tat org.apache.tomcat.dbcp.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)\r\n\tat org.apache.tomcat.dbcp.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)\r\n\t... 26 more\r\n"}]}
2014.06.18 16:34:39 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性
2014.06.18 16:34:39 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:34:40 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-getMemberId-service-unavailable","message":"调用后端服务member.getMemberId抛异常:org.hibernate.exception.SQLGrammarException，服务不可用。\n异常信息:\norg.springframework.dao.InvalidDataAccessResourceUsageException: could not execute query; SQL [select accesspriv0_.id as id1_, accesspriv0_.app_key as app2_1_, accesspriv0_.method_name as method3_1_ from xtraining_db.access_privilege accesspriv0_ where appkey=?]; nested exception is org.hibernate.exception.SQLGrammarException: could not execute query\r\n\tat org.springframework.orm.hibernate3.SessionFactoryUtils.convertHibernateAccessException(SessionFactoryUtils.java:635)\r\n\tat org.springframework.orm.hibernate3.HibernateAccessor.convertHibernateAccessException(HibernateAccessor.java:412)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate.doExecute(HibernateTemplate.java:411)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate.executeWithNativeSession(HibernateTemplate.java:374)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate.find(HibernateTemplate.java:912)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate.find(HibernateTemplate.java:908)\r\n\tat com.xtrainning.hop.dao.AccessPrivilegeDAO.getByAppKey(AccessPrivilegeDAO.java:25)\r\n\tat com.xtrainning.hop.resolver.AccessPrivilegeResolver.findClientPrivilege(AccessPrivilegeResolver.java:14)\r\n\tat com.xtrainning.hop.resolver.AccessPrivilegeResolver$$FastClassByCGLIB$$7abfa3cd.invoke(<generated>)\r\n\tat net.sf.cglib.proxy.MethodProxy.invoke(MethodProxy.java:191)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$CglibMethodInvocation.invokeJoinpoint(Cglib2AopProxy.java:689)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:110)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.AccessPrivilegeResolver$$EnhancerByCGLIB$$fdcf96dc.findClientPrivilege(<generated>)\r\n\tat com.xtrainning.hop.HopServiceAccessController.loadPrivilege(HopServiceAccessController.java:56)\r\n\tat com.xtrainning.hop.HopServiceAccessController.isAppGranted(HopServiceAccessController.java:41)\r\n\tat com.rop.security.DefaultSecurityManager.checkServiceAccessAllow(DefaultSecurityManager.java:249)\r\n\tat com.rop.security.DefaultSecurityManager.validateOther(DefaultSecurityManager.java:127)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:338)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.hibernate.exception.SQLGrammarException: could not execute query\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:90)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.loader.Loader.doList(Loader.java:2231)\r\n\tat org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)\r\n\tat org.hibernate.loader.Loader.list(Loader.java:2120)\r\n\tat org.hibernate.loader.hql.QueryLoader.list(QueryLoader.java:401)\r\n\tat org.hibernate.hql.ast.QueryTranslatorImpl.list(QueryTranslatorImpl.java:361)\r\n\tat org.hibernate.engine.query.HQLQueryPlan.performList(HQLQueryPlan.java:196)\r\n\tat org.hibernate.impl.SessionImpl.list(SessionImpl.java:1148)\r\n\tat org.hibernate.impl.QueryImpl.list(QueryImpl.java:102)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate$30.doInHibernate(HibernateTemplate.java:921)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate$30.doInHibernate(HibernateTemplate.java:1)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate.doExecute(HibernateTemplate.java:406)\r\n\t... 26 more\r\nCaused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'appkey' in 'where clause'\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)\r\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)\r\n\tat java.lang.reflect.Constructor.newInstance(Constructor.java:513)\r\n\tat com.mysql.jdbc.Util.handleNewInstance(Util.java:411)\r\n\tat com.mysql.jdbc.Util.getInstance(Util.java:386)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1053)\r\n\tat com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4096)\r\n\tat com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4028)\r\n\tat com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2490)\r\n\tat com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2651)\r\n\tat com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2734)\r\n\tat com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:2155)\r\n\tat com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:2322)\r\n\tat org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)\r\n\tat org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)\r\n\tat org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:208)\r\n\tat org.hibernate.loader.Loader.getResultSet(Loader.java:1808)\r\n\tat org.hibernate.loader.Loader.doQuery(Loader.java:697)\r\n\tat org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)\r\n\tat org.hibernate.loader.Loader.doList(Loader.java:2228)\r\n\t... 36 more\r\n"}]}
2014.06.18 16:34:40 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性
2014.06.18 16:34:40 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:34:41 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"1","message":"服务不可用","solution":"服务目前无法使用，请保存好错误信息并和服务平台管理员联系","subErrors":[{"code":"isp.member-login-service-unavailable","message":"调用后端服务member.login抛异常:org.hibernate.exception.SQLGrammarException，服务不可用。\n异常信息:\norg.springframework.dao.InvalidDataAccessResourceUsageException: could not execute query; SQL [select accesspriv0_.id as id1_, accesspriv0_.app_key as app2_1_, accesspriv0_.method_name as method3_1_ from xtraining_db.access_privilege accesspriv0_ where appkey=?]; nested exception is org.hibernate.exception.SQLGrammarException: could not execute query\r\n\tat org.springframework.orm.hibernate3.SessionFactoryUtils.convertHibernateAccessException(SessionFactoryUtils.java:635)\r\n\tat org.springframework.orm.hibernate3.HibernateAccessor.convertHibernateAccessException(HibernateAccessor.java:412)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate.doExecute(HibernateTemplate.java:411)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate.executeWithNativeSession(HibernateTemplate.java:374)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate.find(HibernateTemplate.java:912)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate.find(HibernateTemplate.java:908)\r\n\tat com.xtrainning.hop.dao.AccessPrivilegeDAO.getByAppKey(AccessPrivilegeDAO.java:25)\r\n\tat com.xtrainning.hop.resolver.AccessPrivilegeResolver.findClientPrivilege(AccessPrivilegeResolver.java:14)\r\n\tat com.xtrainning.hop.resolver.AccessPrivilegeResolver$$FastClassByCGLIB$$7abfa3cd.invoke(<generated>)\r\n\tat net.sf.cglib.proxy.MethodProxy.invoke(MethodProxy.java:191)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$CglibMethodInvocation.invokeJoinpoint(Cglib2AopProxy.java:689)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:110)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)\r\n\tat org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)\r\n\tat com.xtrainning.hop.resolver.AccessPrivilegeResolver$$EnhancerByCGLIB$$fdcf96dc.findClientPrivilege(<generated>)\r\n\tat com.xtrainning.hop.HopServiceAccessController.loadPrivilege(HopServiceAccessController.java:56)\r\n\tat com.xtrainning.hop.HopServiceAccessController.isAppGranted(HopServiceAccessController.java:41)\r\n\tat com.rop.security.DefaultSecurityManager.checkServiceAccessAllow(DefaultSecurityManager.java:249)\r\n\tat com.rop.security.DefaultSecurityManager.validateOther(DefaultSecurityManager.java:127)\r\n\tat com.rop.impl.AnnotationServletServiceRouter$ServiceRunnable.run(AnnotationServletServiceRouter.java:338)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)\r\n\tat java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:138)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)\r\n\tat java.lang.Thread.run(Thread.java:662)\r\nCaused by: org.hibernate.exception.SQLGrammarException: could not execute query\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:90)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.loader.Loader.doList(Loader.java:2231)\r\n\tat org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)\r\n\tat org.hibernate.loader.Loader.list(Loader.java:2120)\r\n\tat org.hibernate.loader.hql.QueryLoader.list(QueryLoader.java:401)\r\n\tat org.hibernate.hql.ast.QueryTranslatorImpl.list(QueryTranslatorImpl.java:361)\r\n\tat org.hibernate.engine.query.HQLQueryPlan.performList(HQLQueryPlan.java:196)\r\n\tat org.hibernate.impl.SessionImpl.list(SessionImpl.java:1148)\r\n\tat org.hibernate.impl.QueryImpl.list(QueryImpl.java:102)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate$30.doInHibernate(HibernateTemplate.java:921)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate$30.doInHibernate(HibernateTemplate.java:1)\r\n\tat org.springframework.orm.hibernate3.HibernateTemplate.doExecute(HibernateTemplate.java:406)\r\n\t... 26 more\r\nCaused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'appkey' in 'where clause'\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)\r\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)\r\n\tat java.lang.reflect.Constructor.newInstance(Constructor.java:513)\r\n\tat com.mysql.jdbc.Util.handleNewInstance(Util.java:411)\r\n\tat com.mysql.jdbc.Util.getInstance(Util.java:386)\r\n\tat com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1053)\r\n\tat com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4096)\r\n\tat com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4028)\r\n\tat com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2490)\r\n\tat com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2651)\r\n\tat com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2734)\r\n\tat com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:2155)\r\n\tat com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:2322)\r\n\tat org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)\r\n\tat org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)\r\n\tat org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:208)\r\n\tat org.hibernate.loader.Loader.getResultSet(Loader.java:1808)\r\n\tat org.hibernate.loader.Loader.doQuery(Loader.java:697)\r\n\tat org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)\r\n\tat org.hibernate.loader.Loader.doList(Loader.java:2228)\r\n\t... 36 more\r\n"}]}
2014.06.18 16:34:41 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性
2014.06.18 16:34:41 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:34:41 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"21","message":"无效的sessionId参数","solution":"无效的会话,请获取合法的会话"}
2014.06.18 16:39:25 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性
2014.06.18 16:39:25 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:39:26 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"2","message":"开发者权限不足","solution":"不足的ISV权限","subErrors":[{"code":"isv.invalid-permission","message":"权限不够、非法访问"}]}
2014.06.18 16:39:26 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性
2014.06.18 16:39:26 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:39:26 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"2","message":"开发者权限不足","solution":"不足的ISV权限","subErrors":[{"code":"isv.invalid-permission","message":"权限不够、非法访问"}]}
2014.06.18 16:39:26 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性
2014.06.18 16:39:26 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:39:27 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"21","message":"无效的sessionId参数","solution":"无效的会话,请获取合法的会话"}
2014.06.18 16:39:56 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性
2014.06.18 16:39:56 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetMemberIdRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:39:56 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"2","message":"开发者权限不足","solution":"不足的ISV权限","subErrors":[{"code":"isv.invalid-permission","message":"权限不够、非法访问"}]}
2014.06.18 16:39:56 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性
2014.06.18 16:39:56 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.LoginRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:39:56 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"2","message":"开发者权限不足","solution":"不足的ISV权限","subErrors":[{"code":"isv.invalid-permission","message":"权限不够、非法访问"}]}
2014.06.18 16:39:56 [main] DEBUG [com.rop.impl.DefaultRopContext] - 获取com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性
2014.06.18 16:39:56 [main] DEBUG [com.rop.impl.DefaultRopContext] - com.xtrainning.hop.request.mobile.GetProfileRequest不需要签名的属性:[sign, ropRequestContext]
2014.06.18 16:39:56 [main] DEBUG [com.rop.client.DefaultRopClient] - response:
{"code":"21","message":"无效的sessionId参数","solution":"无效的会话,请获取合法的会话"}
